ibis/backends/base/__init__.py

```diff
@@ -2,6 +2,29 @@
 
 from ibis.common.exceptions import TranslationError
 
+from .registry import binary_infix_ops, fixed_arity, operation_registry, unary
```

Comment:
why are you directly importing these here as opposed to from the backends import what is needed?

the reason i ask is that this is common to all backends whereas the registry is for sql only

Reply:
Good point. I was focusing on better structuring the code, and defining what was public, and just did this for simplicity.

What do you think about having a `ibis.backends.base.sql` module where these are made available instead?

Reply:
sure that would be great.